sass/open-edx/_variables.scss

```diff
@@ -1,87 +1,93 @@
-//== edX Bootstrap theme for Bootstrap v4
+// Variables
```

Comment:
I'm wondering if we should have the edX.org theme import this partial, rather than having it essentially duplicated. The only difference between the two should be the theme colors. In fact, maybe we should have a ``_colors.scss`` file for each, and then have a common ``_variables.scss`` that does everything else. I'm not going to review this file other than for colors, since my comments on the edx.org version still apply.

Reply:
There are a few other differences, such as the header size, some of the border radii, and a few other spots I believe. I agree on the breaking up into a _colors.scss file and _variables, but if we do that I think we should break it up into more than just _colors and go with _grids, _elements, etc. For now I will keep as is, unless you feel strongly the other way.